recipes/thermotools/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "thermotools" %}
+{% set version = "0.2.3" %}
+{% set sha256 = "8cd05a705665d9e2da2c1d05b19278945af9927dda7b4edc2c1bf0b1b96d4d71" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy x.x
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - msmtools >=1.1.4
+
+test:
+  imports:
+    - thermotools 
+
+about:
+  home: http://github.com/markovmodel/thermotools
+  license: LGPL 
```

Comment:
Also please [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) to propagate the metadata change.